packages/@aws-cdk/aws-rds/lib/private/parameter-group-family-mapping.ts

```diff
@@ -12,3 +14,28 @@ export interface ParameterGroupFamilyMapping {
    */
   readonly parameterGroupFamily: string
 }
+
+/**
+ * Get the latest parameter group family for this engine. Latest is determined using semver on the engine major version.
+ * When `engineVersion` is specified, return the parameter group family corresponding to that engine version.
+ * Return undefined if no parameter group family is defined for this engine or for the requested `engineVersion`.
+ */
+export function calculateParameterGroupFamily(
```

Comment:
I'm having a hard time reconciling this with #9016. In the other PR, parameterGroupFamilies look to be completely removed; will this function be deleted then?

Reply:
Yep. In that PR, the major version is set explicitly for a specific version (instead of being calculated).